### DISCUSSION TITLE
Interactive task list

### DISCUSSION THREAD

**Discussion by taszap** [GitbookIO/community#6] [Feature Requests]
### What's your feature/idea?

Is it possible to make the Task List boxes selectable? I was hoping that since selecting Task List those boxes would be something that can be checked and when you refresh the page they get wiped or something along the lines of clearing out cookies etc. 

### What's your desired outcome?

I want the Task list to be useful to readers, not just editors. I would like each reader to be able to interact with it and tick/untick the tasks and have a view individual to them. 

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [ ] Medium
- [X] Low

**Comment by amenity** [comment]
I'm pretty sure that task lists used to be interactive for readers. We use [this one](https://atd-dts.gitbook.io/wiki/onboarding/welcome-to-dts) to onboard our new employees, and it has always been useful that people can start using it before all their accounts (including Github/Gitbook) are set up. 

When and why was this feature retired?

**Reply by taszap** [reply]
Hi @amenity the task list was never interactive to readers I'm afraid. 
Your employees could have checked or unchecked it if they had editor access.

**Reply by amenity** [reply]
I think this needs to be double-checked, we definitely have used this functionality before. In any case, it is a should-have for us.
